inst/eDISH_app/server.R

```diff
@@ -28,37 +28,84 @@ function(input, output, session){
   #
   # reutrns updated settings and validation status
     settings_new <-   callModule(renderSettings, "settingsUI",
-                                # data = isolate(reactive(dataUpload_out$data_selected())),  # this doesnt make sense
                                  data = reactive(dataUpload_out$data_selected()),
                                  settings = reactive(dataUpload_out$settings()),
                                  status = reactive(dataUpload_out$status()))
 
 
   # update settings navbar
     output$settings_tab_title = renderUI({
-      if (settings_new$status()$valid==TRUE){
+      num_valid <- settings_new$status() %>% flatten() %>%  keep(., names(.)=="valid") %>% unlist() %>% sum()
```

Comment:
I'm leaning towards removing the icon from the settings navbar and just showing the chart status ...

Reply:
Done.